Traceback (most recent call last):
  File "/home/ceriotti/lavoro/code/i-pi-cosmo/bin/i-pi-py_driver", line 8, in <module>
    from pes import *
  File "/local/home/ceriotti/lavoro/code/i-pi-cosmo/drivers/py/pes/__init__.py", line 14, in <module>
    module = importlib.import_module("." + module_name, __package__)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/local/home/ceriotti/lavoro/code/i-pi-cosmo/drivers/py/pes/metatensor.py", line 15, in <module>
    import metatensor.torch
  File "/home/ceriotti/local/lib/python3.10/site-packages/metatensor/torch/__init__.py", line 5, in <module>
    from ._c_lib import _load_library
  File "/home/ceriotti/local/lib/python3.10/site-packages/metatensor/torch/_c_lib.py", line 14, in <module>
    if not version_compatible(metatensor.__version__, BUILD_METATENSOR_CORE_VERSION):
AttributeError: module 'metatensor' has no attribute '__version__'
